packaging/debian/postinst

```diff
@@ -29,28 +29,12 @@ DAEMON_USER=storj-storagenode
 case "$1" in
     configure)
 	installed_version="$2"
-	if [ $installed_version != "" ]; then
-	    echo "This is a reconfiguration or a reinstallation"
-	fi
-	db_get storagenode/identitydir
-	identitydir="$RET"
-	db_get storagenode/wallet
-	wallet="$RET"
-	db_get storagenode/email
-	email="$RET"
-	db_get storagenode/external_addr
-	external_addr="$RET"
-	db_get storagenode/storagedir
-	storagedir="$RET"
-	db_get storagenode/old_storagedir
-	old_storagedir="$RET"
-	db_get storagenode/diskspace
-	diskspace="$RET"
-	db_get storagenode/bandwidth
-	bandwidth="$RET"
+	#if [ $installed_version != "" ]; then
+	#    echo "This is a reconfiguration or a reinstallation"
+	#fi
 
 	# TODO: should not download binaries if it's only a reconfigure
-	echo "Downloading binaries..."
+	#echo "Downloading binaries..."
 	for binary in "storagenode" "storagenode-updater"; do
 	    wget -O /tmp/${binary}.zip ${BINARIES_SERVER}/${binary}_linux_amd64.zip
```

Comment:
I mentioned this in #7, but again just in case:

We should never write to a file with a predictable name in `/tmp`. We should use `mktemp` if necessary. I know it's not part of this PR exactly, but it's important.

Reply:
I will do :)